src/Compilers/CSharp/Portable/Symbols/Source/SourceEventSymbol.cs

```diff
@@ -484,7 +484,7 @@ protected void CheckModifiersAndType(DiagnosticBag diagnostics)
             }
             else if (IsAbstract && IsVirtual)
             {
-                diagnostics.Add(ErrorCode.ERR_AbstractNotVirtual, location, this);
+                diagnostics.Add(ErrorCode.ERR_AbstractNotVirtual, location, this.GetAttributeTarget().ToString().ToLower(), this);
```

Comment:
Changed it to this.Kind.Localize() (Localized and has all values we need in this case)